Gemfile

```diff
@@ -46,11 +46,14 @@ gem 'recurring_select'
 gem 'roo', '~> 2.0.0'
 gem 'roo-xls'
 gem 'spreadsheet'
+gem 'forem', :github => "radar/forem", :branch => "rails4"
```

Comment:
Please add this just above the `gem 'foodsoft_forum'` line with a comment like with acts_as_versioned, because it's specific to this plugin.